src/fragments/forms/settings/i18n/settings.i18n.cs-cz.js

```diff
@@ -0,0 +1,50 @@
+
+export default {
+  settings: {
+    endpointsSectionTitle: 'Koncové body služby (URI nebo úplná adresa URL)',
+    language: 'Jazyk',
+    appLocale: 'Lokalizace aplikace',
+    unit: 'Jednotka',
+    areaUnit: 'Jednotka plochy',
+    saveToLocalStorage: 'Uložit v mém prohlížeči',
+    routeInstructions: 'Instrukce pro navigaci',
+    randomizedIsochroneColors: 'Náhodné izochronní barvy',
+    distanceMarkers: 'Značky vzdálenosti',
+    elevationProfile: 'Profil převýšení',
+    mapSettings: 'Zobrazení mapy a další informace',
+    defaultMapSettingsRestored: 'Obnovení a uložení výchozího nastavení mapy',
```

Comment:
Automatic translation suggests "Restore and save default map settings" which is an active formulation of the originally passive "Default map settings restored and saved".
Context here is the "Restore default"-button in the settings, this is the message shown to users once this has been clicked.
Is the formulation in czech to inform users as the english one is?